components/automate-gateway/api/iam/v2/tokens.proto

```diff
@@ -13,6 +14,26 @@ import "components/automate-grpc/protoc-gen-policy/api/annotations.proto";
 import "components/automate-grpc/protoc-gen-policy/iam/annotations.proto";
 
 service Tokens {
+
+  /*
+  Create a token
+
+  Creates a token.
+  Name, ID are required.
+  Active defaults to true when not specified.
+  Value is auto-generated when not provided.
+  ```
+  {
+    "name": "token 1",
+    "id": "token-1",
+    "active": true,
+    "projects": [
+      "east-region",
+      "west-region"
+    ]
```

Comment:
Sorry to harp on a theme: we have examples for these token endpoints but the policy endpoints show no examples. Would suggest to go the route of no examples, because the doc generator automatically provides a sample (without data, but I think it is sufficient and saves all the maintenance burden).